src/components/notice/_notice.scss

```diff
@@ -0,0 +1,52 @@
+// Notice, text with an icon to the left
+.gv-c-notice {
+  @include clearfix;
+
+  position: relative;
+}
+
+.gv-c-notice__icon {
+  position: absolute;
+  top: 50%;
+  left: 0;
+  margin-top: -17px; // Half the height of the important icon
+
+  display: inline-block;
+
+  background-repeat: no-repeat;
+  background-position: 0 0;
+}
+
+.gv-c-notice__icon--important {
+  width: 35px;
+  height: 35px;
+
+  background-image: file-url("toolkit/icon-important.png");
+
+  @include device-pixel-ratio {
+    background-image: file-url("toolkit/icon-important-2x.png");
+    background-size: 100%;
+  }
+}
+
+.gv-c-notice__icon-fallback-text {
+  position: absolute;
```

Comment:
What are we doing with `visually-hidden`? Do we at least want a mixin for it for use in places like this?